2013/05/31 00:28:00 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/r3ntg3n/www/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/r3ntg3n/www/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/r3ntg3n/www/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/r3ntg3n/www/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/r3ntg3n/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/r3ntg3n/www/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/r3ntg3n/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/r3ntg3n/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/r3ntg3n/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/r3ntg3n/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/r3ntg3n/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/r3ntg3n/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/r3ntg3n/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/r3ntg3n/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/r3ntg3n/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/r3ntg3n/www/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/r3ntg3n/www/yii/delivery/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/delivery/gii/model
HTTP_REFERER=http://yii/delivery/gii/default/index
---
